app/build.gradle.kts

```diff
@@ -231,6 +232,11 @@ dependencies {
     implementation("androidx.work:work-runtime:2.9.0")
     implementation("androidx.work:work-runtime-ktx:2.9.0")
     implementation("com.github.Blatzar:NiceHttp:0.4.11") // HTTP Lib
+
+    // Baseline Profile and Automation
```

Comment:
testImplementation?

Reply:
no they are not test implementation, the profiler generates the profile and and the module is stored with apk, in assets/dexopt. if we rename to test then probably it will fail even if succeds the baseline profile wont be included in app